test/assigned_fulfillment_order_test.rb

```diff
@@ -14,7 +14,7 @@ class AssignedFulFillmentOrderTest < Test::Unit::TestCase
 
         assert_equal 2, assigned_fulfillment_orders.count
         assigned_fulfillment_orders.each do |fulfillment_order|
-          assert_equal 'ShopifyAPI::FulfillmentOrder', fulfillment_order.class.name
+          assert fulfillment_order.is_a?(ShopifyAPI::FulfillmentOrder)
```

Comment:
nit: should probably move this into the other PR dealing with assigned fulfillment order.